### ISSUE TITLE
Spill zone(s) support

### ISSUE THREAD

**Issue by amitkeret** [nathanvdh/homeassistant-airtouch2#13]
Thank you for making this great integration!
Would be a great addition if this integration could also display Spill zones and alerts.
I believe the python library [already supports this](https://github.com/search?q=repo%3Anathanvdh%2Fairtouch2-python+spill&type=code)?

**Comment by nathanvdh** [comment]
Correct, the support for this is there in the library. I'm just not sure how/where to display this info in home assistant. Do you have any ideas?

**Comment by TETZUO** [comment]
Maybe just a on/off binary sensor? That we way people can do what they like with it.
________________________________

**Comment by amitkeret** [comment]
Thanks for the quick reply!

I would agree with @TETZUO.
Since HA's `climate` and `fan` integrations do not have a native "spill" attribute, the best would be to add a binary sensor.

Ideally, a spill sensor would be added to each individual zone device, as well as to the overall AC (climate) device.
This is because some systems have spill alert for individual zones, while others (like mine) have a "general" spill indicator for the entire system.
